test/videojs-http-streaming.test.js

```diff
@@ -250,9 +254,9 @@ QUnit.test('autoplay seeks to the live point after playlist load', function(asse
   let currentTime = 0;
 
   this.player.autoplay(true);
-  this.player.on('seeking', () => {
-    currentTime = this.player.currentTime();
-  });
+  this.player.tech_.currentTime = (ct) => {
```

Comment:
During `seeking` currentTime is not actually set to the final value in ie 11 and edge. So instead we get currentTime from mocking the tech method.